prov/rxm/src/rxm_cq.c

```diff
@@ -534,7 +569,8 @@ static ssize_t rxm_handle_rndv(struct rxm_rx_buf *rx_buf)
 		ret = rxm_msg_mr_regv(rx_buf->ep,
 				      rx_buf->recv_entry->rxm_iov.iov,
 				      rx_buf->recv_entry->rxm_iov.count,
-				      total_recv_len, FI_READ, rx_buf->mr);
+				      total_recv_len, FI_READ | FI_REMOTE_WRITE,
```

Comment:
here and other places, we'll want this set based on enabling writes for rdvz.  and here we may not need FI_READ in that case.  You may want a global to set these, maybe in the ops structure that's used for read vs write.